docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Automating Your Data Pipelines | Dagster
+description: This guide illustrates how to automate your data pipelines
+---
+
+# Automating your data pipelines
+
+In this guide, we’ll walk you through how to run your data pipelines without manual intervention, i.e. automate, and identify the Dagster tools to make that happen.
+
+This guide assumes you have some familiarity with several Dagster concepts, including [software-defined assets](/concepts/assets/software-defined-assets) and [jobs](/concepts/ops-jobs-graphs/jobs).
+
+Dagster offers a few different ways of automating data pipelines, and choosing the right Dagster tool depends on your specific needs. In this guide we'll walk through a few different cases, and which Dagster tool you should use in each case.
+
+---
+
+## My data should be updated every morning
+
+You want some data to be updated every day by a certain time to refresh a dashboard, have updated data go to to a system or you want your database to have the most recent information for the team to work with. This is one of the most traditional cases when building a pipeline. Lets go into how to do this with Dagster!
+
+### I want a job to start at a specific time
+
+In Dagster, we have [basic scheduling](/concepts/partitions-schedules-sensors/schedules#basic-schedules) capabilities, which means that you say how often and when you want a job to run, and it will run. This can mean daily, weekly, or hourly. Dagster also supports any type of [cron scheduling](https://en.wikipedia.org/wiki/Cron).
+
+```python file=/guides/dagster/automating_pipelines/vanilla_schedule.py
+from dagster import AssetSelection, define_asset_job, ScheduleDefinition
+
+asset_job = define_asset_job("asset_job", AssetSelection.groups("some_asset_group"))
+
+basic_schedule = ScheduleDefinition(job=asset_job, cron_schedule="0 9 * * *")
+```
+
+You can also use the <PyObject object="schedule" decorator /> decorator if you want to provide custom run config and tags. For example, if you want to pass a parameter to the job at runtime, for example, activity = ‘party’ on the weekend and activity=’grind’ on the weekdays, you can use the configuration to pass parameters in.
```

Comment:
```suggestion
You can also use the <PyObject object="schedule" decorator /> decorator if you want to provide custom run config and tags. For example, if you want to pass a parameter to the job at runtime, for example, `activity = ‘party’` on the weekend and `activity=’grind’` on weekdays, you can use the configuration to pass parameters in.
```

🔧 Formatting these as code so they're easier to spot and be interpreted as literal values